b2share/config.py

```diff
@@ -47,6 +41,180 @@
     account_json_loader, account_json_patch_loader,
 )
 
+from invenio_search.config import SEARCH_ELASTIC_HOSTS
+
+SEARCH_ELASTIC_HOSTS = []
+
+for elastic_host in os.environ.get("ELASTIC_HOSTS", "localhost:9200").split(','):
+    try:
+        (host, port) = elastic_host.split(':')
+    except:
+        host = elastic_host
+        port = 9200
+
+    SEARCH_ELASTIC_HOSTS.append(dict(host=host, port=port, use_ssl=False, ssl_show_warn=False))
```

Comment:
Invenio-Search package needs a list for it's elasticsearch hosts, but ELASTIC_HOSTS which is the envvar used by elasticsearch-py, uses string which contains comma-separated list of hosts.

We could not find how B2SHARE V2 actually handles the configuration for this. docker-compose.yml in dockerize-repo has `B2SHARE_SEARCH_ELASTIC_HOSTS`  but this is not referenced in B2SHARE v2 code.